app/js/actions/labOrders/labOrderableAction.js

```diff
@@ -0,0 +1,35 @@
+import * as types from '../actionTypes';
+import axiosInstance from '../../config';
+import loading from '../loading';
+
+export const fetchLabOrderablesSuccess = orderables => ({
+  type: types.FETCH_LAB_ORDERABLE_SUCCESS,
+  orderables,
+});
+
+export const fetchLabOrderablesFailure = error => ({
+  type: types.FETCH_LAB_ORDERABLE_FAILURE,
+  error,
+});
+
+/**
+ * @function
+ * @returns {Object} - Object consisting lab orderables from API resource
```

Comment:
A little bit more effort is required in this JSDoc documentation 🙂.
The documentation states that the `function` below is returning an`object` whereas it is actually returning  a `function`.

Reply:
Thanks for the clarification.